forum/qa-include/pages/admin/admin-default.php

```diff
@@ -899,7 +899,6 @@
 				echo qa_html($message).str_repeat('    ', 1024)."<br>\n";
 
 				flush();
-				sleep(1);

```

Comment:
W tej chwili nie mam pewności bo nie mam odpalonego kodu, ale skąd założenie, że bezpieczne jest usunięcie tego sleepa? W zasadzie nie mam pojęcia w jakim celu został tu wstawiony przez twórcę, ale skoro wstawił to musiał mieć jakiś cel - czy da się sprawdzić funkcjonowanie tego forum tak, żeby sprawdzić to miejsce?

Reply:
Z tego co wiem, ten plik to jest strona startowa panelu administratora (ta co się włącza od razu). Ja po prostu przez pół godziny łaziłem po różnych stronach forum, zadawałem pytania, relogowałem się, odpowiadałem i żadnego błędu nie było.

Reply:
A konsola w chrome nic nie pokazywała? A nie pokazało się nic w logach w var/log? W tej chwili nie mam jak opalić tego kodu - stąd takie pytanie :)

Reply:
Szczerze, nie pamiętam, ale raczej nie - inaczej nie przepuściłbym tego.